acdh_cidoc_pyutils/__init__.py

```diff
@@ -119,6 +119,8 @@ def create_e52(
     label=True,
     not_known_value="undefined",
     default_lang="en",
+    time_span_type=None,
```

Comment:
* should be an URIRef object, because now we hard code parts of the URI scheme for the time_span_type
* by passing in a full URI we can also skip the `domain` param